docs/SOPs/data_retraction_SOP.md

```diff
@@ -126,7 +126,19 @@ The metadata spreadsheet can be in many different locations. Please make sure yo
    1. Head to the [Wrangler team Google group](https://groups.google.com/a/data.humancellatlas.org/g/wrangler-team) and delete any email thread that may contain the spreadsheet.
    1. Send an email to the <a href="mailto:wrangler-team@data.humancellatlas.org?subject=[URGENT]%20Dataset%20retraction%20DATASET%20SHORTNAME&body=Hello%0A%0AWe%20have%20been%20asked%20to%20retract%20the%20dataset%20%20%22%3Cshortname_of_dataset%3E%22.%20%0A%0APlease%20delete%20your%20own%20copies%20of%20the%20following%20emails%3A%0A%0A-%20%3Ctitle_of_email_thread%3E%0A%28more%20if%20necessary%29%0A%0APlease%20remove%20any%20local%20copies%20you%20may%20have%20of%20spreadsheets%20associated%20with%20%22%3Cshortname_of_dataset%3E%22%0A%0AIf%20you%20were%20involved%20in%20wrangling%20this%20dataset%2C%20please%20ensure%20no%20local%20copies%20of%20the%20data%2Fmetadata%20are%20left%20in%20the%20EC2%20instance%0A%0AMany%20thanks%20for%20your%20cooperation%0A%0ABest%20regards%2C%0A%0A%3CWrangler%20name%3E">wrangler team</a>
    1. Delete your own local copies of the thread in your mail apps.
```

Comment:
What about backup emails? Do IT services have these?

Reply:
Good question, I have no idea

Reply:
I just checked and apparently they keep a copy for 52 weeks, but I think backups are not a problem as long as they are safely stored and we are sure will be deleted. @lauraclarke ?